### DISCUSSION TITLE
Thank you and Happy New Year !

### DISCUSSION THREAD

**Discussion by brenuart** [logfellow/logstash-logback-encoder#482] [General]
Hi Phil,

Just wanted to say a big thank you for this amazing library! It works like a charm and is in use in all of our projects. The documentation is clear and concise and I love how the code is written and organised. Thanks for your work and effort during all these years!

We wish you a happy new year and all the best for next year (covid-free, what else?)

/bertrand

PS: feel free to close this issue after reading

**Comment by philsttr** [comment]
Thanks Bertrand!

This message means a lot, and makes the time that I put into this more fulfilling.  I really appreciate it.

Wish you the best for the new year as well!
